Tests/Legacy/Shopware/Connect/SDKTest.php

```diff
@@ -111,6 +114,12 @@ public function testHandleProductUpdates()
 
     public function testExportProductWithPurchasePrice()
     {
+
+        Shopware()->Db()->executeQuery(
```

Comment:
why this is needed? Because of some dependency on previous tests ?

Reply:
yes exactly this row was previously inserted by another integration test.